### PR TITLE
Rephrasing

### PR DESCRIPTION
**Pull request by nevrome**
I realized that the schema text is a bit chaotic. I think we can rephrase it a bit to make it clearer and more uniform. A version change is imho not required.

### CODE REVIEW

**Comment by nevrome** [comment]
For the review I suggest you don't look so much at the line-wise differences, because it's confusing in this case, but focus on the new document version here: https://github.com/poseidon-framework/poseidon-schema/blob/rephrasing2.7.1/README.md

**Comment by TCLamnidis** [comment]
I have often seen the validation keywords put in all caps (e.g. `A package therefore MUST contain:`). I generally find that this system makes it clearer to a reader that the specific words used matter and it's not just about what sounds nicer.

File: README.md [hunk 1]
```diff
@@ -1,64 +1,70 @@
...
+A changelog for this standard is available on the website [here](https://poseidon-framework.github.io/#/changelog).
 
-## The Poseidon package structure
+### The Poseidon package structure
 
-All ancient and modern data in Poseidon are distributed into so-called packages, which are directories containing a dedicated set of files. Packages correspond to published sets of genomes, or in case of unpublished projects, ongoing (and growing) sets of samples currently analysed. All text files in the package are UTF-8 encoded.
+A Poseidon package stores genotype data with context information for DNA samples from (ancient) (human) individuals. Packages are defined by the POSEIDON.yml file, which holds relative paths to all other files in a package.
 
-Every package should have the following files: 
+A package therefore must contain:
```
Comment (TCLamnidis) [inline README.md]: See my comment above. I would all-caps the keywords `MUST`, `CAN/COULD`, `SHOULD`

File: README.md [hunk 2]
```diff
@@ -1,64 +1,70 @@
...
+Switzerland_LNBA_Roswita/Switzerland_LNBA.fam
 Switzerland_LNBA_Roswita/Switzerland_LNBA.janno
 Switzerland_LNBA_Roswita/Switzerland_LNBA.ssf
 Switzerland_LNBA_Roswita/Switzerland_LNBA.bib
 Switzerland_LNBA_Roswita/README.md
 Switzerland_LNBA_Roswita/CHANGELOG.md
 ```
 
-## The `POSEIDON.yml` file
+All text files in the package are UTF-8 encoded.
```
Comment (TCLamnidis) [inline README.md]: Is this a requirement, or are you simply stating that is the case here? If the former, this should probably be phrased with a MUST/SHOULD kind of statement.

File: README.md [hunk 3]
```diff
@@ -76,40 +82,40 @@ readmeFile: README.md
...
 - The decimal separator for all floating point numbers is `.`.
 
-For a more extensive documenation of the columns and their interaction see [https://poseidon-framework.github.io/#/janno_details](https://poseidon-framework.github.io/#/janno_details).
+For a more extensive documentation of the columns and their interaction see [https://poseidon-framework.github.io/#/janno_details](https://poseidon-framework.github.io/#/janno_details).
 
-## The `.bib` file
+### The `.bib` file
 
-[BibTeX](http://www.bibtex.org/) file with all references listed in the `.janno` file. The bibtex keys must fit to ones used in the `.janno` file.
+A [BibTeX](http://www.bibtex.org/) file with all references listed in the `.janno` file. The entry keys must fit to the ones used in the `.janno` file.
```
Comment (TCLamnidis) [inline README.md]:
```suggestion
A [BibTeX](http://www.bibtex.org/) file with all references listed in the `.janno` file. The entry keys must fit the ones used in the `.janno` file.
```

**Review by TCLamnidis — APPROVED** [review]
Minor comments regarding phrasing of keywords. one minor change of phrase.
LGTM!

File: README.md (hunk 1 above)
Comment (nevrome) [reply README.md]: Good point. I added that.

File: README.md (hunk 2 above)
Comment (nevrome) [reply README.md]: Done. This is a MUST.

**Review by stschiff — APPROVED** [review]
Nice, good to go.

**Pull request merged**